pages/Iron-Dust.md

```diff
@@ -18,3 +18,9 @@ It is also used in the following [alloys](https://github.com/TheBusyBiscuit/Slim
 * [Gilded Iron](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Gilded-Iron)
 * [Nickel Ingot](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Nickel-Ingot)
 * [Cobalt Ingot](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Cobalt-Ingot)
+
+... and directly in creating other items:
+
+* [Magnet](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Magnet)
+
+Iron is one of the most used dusts in SlimeFun recipes, either as dust or as [Iron Ingot](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Iron-Ingot).
```

Comment:
Why have a page for Iron Ingots? That's a vanilla item.

Reply:
Yeah, lol,  it was there before, hasn't even touched it ('penta' created it). I think it's good to have as iron ingots are used in a lot of important recipes and to help discover how to create vanilla Iron ingots via SlimeFun. There is a clear indication in the article that this is a vanilla item :)

Reply:
Huh, alright then.